packages/cli/src/Mfa/totp.service.ts

```diff
@@ -20,7 +20,7 @@ export class TOTPService {
 		}).toString();
 	}
 
-	verifySecret({ secret, token, window = 1 }: { secret: string; token: string; window?: number }) {
+	verifySecret({ secret, token, window = 2 }: { secret: string; token: string; window?: number }) {
```

Comment:
This change now allows the latest TOTP token, as well as the the token prior to that, which helps against occasional timing issues.